sailbot_ws/src/sailbot/sailbot/battery_monitor.py

```diff
@@ -0,0 +1,63 @@
+import rclpy
+from rclpy.node import Node
+from std_msgs.msg import String
+import RPi.GPIO as GPIO
+
+class BatteryMonitor(Node):
+
+    def __init__(self):
+        super().__init__('battery_monitor')
+        self.publisher_ = self.create_publisher(String, 'battery_status', 10)
+        timer_period = 5  # seconds
+        self.timer = self.create_timer(timer_period, self.timer_callback)
+        # Pin Definitions Need 5 board pins next to one another
+        self.bat_p1 = 26  # 14V pin
+        self.bat_p2 = 13  # 13V pin
+        self.bat_p3 = 6  # 12V pin
+        self.bat_p4 = 5  # 11V pin
+
+        # Pin Setup:
+        # Board pin-numbering scheme
+        GPIO.setmode(GPIO.BCM)  # Set pins low for high inputs
+        GPIO.setup(self.bat_p1, GPIO.IN)  # + 14V
+        GPIO.setup(self.bat_p2, GPIO.IN)  # + 13V
+        GPIO.setup(self.bat_p3, GPIO.IN)  # + 12V
+        GPIO.setup(self.bat_p4, GPIO.IN)  # + 11V
+
+    def timer_callback(self):
+        msg = String()
+        # #If Else Statements
+        if GPIO.input(self.bat_p1):
+            msg.data = "+14V you're balling"
```

Comment:
As Tony mentioned in the lab, it might be wise to just send an integer representing the state or a voltage value in the message data. These longer messages should probably be displayed in the UI where applicable.